integration/test_storage.py

```diff
@@ -93,7 +93,7 @@ def test_put(self):
         self.assertEqual(p.description, new_desc)
 
     # can't test this as patch doesn't seem to work for storage pools.
-    # Need to wait until bug: https://github.com/lxc/lxd/issues/4709
+    # Need to wait until bug: https://github.com/canonical/lxd/issues/4709
```

Comment:
Looks like this has been fixed? Maybe a follow up PR can unskip this test :smile:

Reply:
Yep, those were meant to be addressed in a subsequent PR ;)

Reply:
Okey dokey!